src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/CommitManager.cs

```diff
@@ -62,6 +64,12 @@ internal CommitManager(ImmutableArray<char> potentialCommitCharacters, RecentIte
                 && excludedCommitCharacter.Contains(typedChar));
         }
 
+        private ImmutableHashSet<char> GetPotentialCommitCharacters()
+            => _textView.Properties.TryGetProperty(CompletionSource.PotentialCommitCharacters, out ImmutableHashSet<char> potentialCommitCharacters)
```

Comment:
Stylewise, I admit I'm not sure I see the value of a => syntax when it's a four line ternary operator inside of it. :smile: